src/main/java/org/alfresco/repo/content/transform/TikaPoweredContentTransformer.java

```diff
@@ -223,6 +225,14 @@ protected ParseContext buildParseContext(Metadata metadata, String targetMimeTyp
        }
        return context;
     }
+    
+    protected void buildExtraOptionsforTika(TransformationOptions options)
+    {
+        if (options == null)
+        {
+            options = new TransformationOptions();

```

Comment:
There is something wrong here. The newly created object is discarded. I think you need to return it from the method and then on the call do:

options = buildExtr...(options);